contrib/sssd.spec.in

```diff
@@ -764,14 +764,14 @@ done
 %attr(700,%{sssd_user},%{sssd_user}) %dir %{dbpath}
 %attr(775,%{sssd_user},%{sssd_user}) %dir %{mcpath}
 %attr(700,root,root) %dir %{secdbpath}
-%attr(751,root,root) %dir %{deskprofilepath}
+%attr(751,sssd,sssd) %dir %{deskprofilepath}
```

Comment:
not `%{sssd_user}` ?

Reply:
The same mistake again *facepalm*

Reply:
Thanks, fixed.

Reply:
> The same mistake again _facepalm_

Ah, that's because I was looking at old commit d163a120b922a49b458dc9568d90c4066cee2d73 where diff was like
```
- %attr(751,sssd,sssd) %dir %{deskprofilepath}
+ %attr(751,root,root) %dir %{deskprofilepath}
```